CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -525,6 +531,12 @@ namespace Kartaclysm
 		return glm::quat(glm::vec3(0.0f, m_fDirection, 0.0f));
 	}
 
+	float ComponentKartController::GetRotationMod()
+	{
+		float m_fSpin = m_fSpinout * m_fSpinFactor;
```

Comment:
This should not have prefix m_ if it is locally declared. It could also be condensed into one line instead of creating a variable.